src/SampleViewExtension/SampleWindowViewModel.cs

```diff
@@ -10,7 +11,7 @@ public class SampleWindowViewModel : NotificationObject, IDisposable
         private string selectedNodesText = "Begin selecting ";
         private ReadyParams readyParams;
 
-        public string SelectedNodesText = @"There are {readyParams.CurrentWorkspaceModel.Nodes.Count()} nodes in the workspace.";
+        public string SelectedNodesText => $"There are {readyParams.CurrentWorkspaceModel.Nodes.ToList().Count()} nodes in the workspace.";
```

Comment:
@Racel This is still using C# 6.0.  @mjkkirschner was suggesting to use String.Format instead such as
```
String.Format("There are  {0} nodes in the workspace.", readyParams.CurrentWorkspaceModel.Nodes.ToList().Count());
```
I originally suggested using a helper function because I assumed for your project you would have to iterate through the active nodes for additional information (such as node positions).

Reply:
@alfarok - FYI, regarding the approach you mentioned above. I think you may not be able to use one instance variable to set another instance variable. I get an error when "field initializer cannot reference the non-static field..."

Reply:
There are a few different possible solutions.  You could add a getter to `SelectedNodesText`

@mjkkirschner What do you think?

Reply:
a couple things...
if we're not using c#6 features then you can't use the => for the field initializer. I wouldn't do this anyway.

Just make it a property or empty field and set it in the constructor.

Reply:
@Racel and I spoke briefly yesterday,  I believe she was using `=` above.  That solution makes sense though, thanks.